op-service/sources/l1_client.go

```diff
@@ -66,10 +70,23 @@ func NewL1Client(client client.RPC, log log.Logger, metrics caching.Metrics, con
 		return nil, err
 	}
 
+	var clientToUse EthClientInterface
+
+	if config.PrefetchingTimeout > 0 && config.PrefetchingWindow > 0 {
```

Comment:
```suggestion
	if config.PrefetchingTimeout > 0 || config.PrefetchingWindow > 0 {
```
I think if either is 0, prefetching wouldn't work

Reply:
@sebastianst Wait, I think we want `&&` here, no? In other words "Only use `NewPrefetchingEthClient` if both a `PrefetchingTimeout` and `PrefetchingWindow` are provided." Using `||` here would allow a user to set e.g. a window but no timeout, or a timeout but no window, and pass it down to `NewPrefetchingEthClient`.

Reply:
sorry, I misread 🙈 of course, I thought this was the other case. sorry!